contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -88,10 +95,22 @@ contract RaidenMicroTransferChannels {
         challenge_period = _challenge_period;
     }
 
+    /// @dev Sets the address for the latest contract version
+    /// @param _latest_version_address The address for the latest contract version.
+    function setLatestVersionAddress(address _latest_version_address) public isOwner {
+        require(addressHasCode(_latest_version_address));
+        latest_version_address = _latest_version_address;
+    }
+
     /*
      *  Public helper functions (constant)
      */
 
+    /// @dev Returns the address for the latest contract version.
+    function getLatestVersionAddress() public view returns(address) {
```

Comment:
Why a getter? It should be constant. Similar to my comment above.

Reply:
It gets the address for the last version contract, not the version itself.